assets/src/edit-story/migration/migrate.js

```diff
@@ -22,12 +22,14 @@ import dataPixelTo1080 from './migrations/v0002_dataPixelTo1080';
 import fullbleedToFill from './migrations/v0003_fullbleedToFill';
 import squareToShape from './migrations/v0004_squareToShape';
 import mediaElementToResource from './migrations/v0005_mediaElementToResource';
+import tmpShapeBgToResource from './migrations/v0006_tmpShapeBgToResource';
```

Comment:
The `v000X` and `x: [...]` below should be the same. Just for ease of searching. Also, I think this now needs to be rebased. But otherwise, see the notes above on resources and shapes :)